src/components/CopyUrlBox.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
```

Comment:
lmao i have a copy url component that is still waiting to get reviewed xd

Reply:
omfg LOL MB